evm-contracts/src/v0.6/dev/FluxAggregator.sol

```diff
@@ -592,12 +592,34 @@ contract FluxAggregator is AggregatorInterface, Owned {
     updateAvailableFunds();
   }
 
+  function asdf(address _oracle, uint32 _queriedRoundId) private view returns (bool _eligible, uint128 _state) {
```

Comment:
Is this still needed? If so, can we give it a better name?

Reply:
Whoops, good catch.